packages/providers/fal/image-to-image/fal_image_to_image/models/base.py

```
@@ -8,24 +8,26 @@
 import fal_client
 
 from ..utils.file_utils import download_images, ensure_output_directory
-from ..config.constants import MODEL_ENDPOINTS, MODEL_DISPLAY_NAMES
+from ai_content_pipeline.registry import ModelRegistry
+import ai_content_pipeline.registry_data  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)